dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ExclusionFileParser.java

```diff
@@ -62,12 +68,29 @@
   static ImmutableList<LinkageErrorMatcher> parse(Path exclusionFile)
       throws SAXException, IOException {
 
+    File exclusion = exclusionFile.toFile();
```

Comment:
why convert to a File? Can we stick to Paths all the way through?

Reply:
Updated to use Files.newInputStream.